vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -36,6 +36,7 @@ namespace Microsoft.ReactNative {
     UInt16 DebuggerPort { get; set; };
     IRedBoxHandler RedBoxHandler { get; set; };
     IReactDispatcher UIDispatcher { get; set; };
+    IReactDispatcher JSDispatcher { get; set; };
```

Comment:
Is it intentional that we're exposing the JS thread as being user configurable?

Reply:
I get it removed